x-pack/plugin/ml/qa/single-node-tests/src/test/java/org/elasticsearch/xpack/ml/transforms/PainlessDomainSplitIT.java

```diff
@@ -355,4 +366,16 @@ public void testHRDSplit() throws Exception {
             fail("Anomaly records were not found within 5 seconds");
```

Comment:
This `fail` dates back to when the `assertBusy` on line 330 was an `awaitBusy`.  I think the whole `try`/`catch` is redundant now.

Reply:
For sure, I will refactor the test to make it cleaner